### ISSUE TITLE
Merge identical alleles when loading

### ISSUE THREAD

**Issue by kimrutherford** [pombase/pombase-chado#577]
As an example, we have multiple deletion alleles for some genes.  They should be merged when loading.  For an example run:

```
select f.name, f.uniquename, p.value from feature f
  join featureprop p on f.feature_id = p.feature_id where name like '%tsc1%';
```

Another example from Midori:
  http://pombase2.bioinformatics.nz/genotype/c1136816cdaa2d17-genotype-1
vs
  http://pombase2.bioinformatics.nz/genotype/6998329e8f9ad0bc-genotype-1 

We also need to merge identical genotypes once the alleles are merged.

**Comment by mah11** [comment]
not just deletion alleles, as noted on the linked website ticket

**Comment by kimrutherford** [comment]
> not just deletion alleles, as noted on the linked website ticket

I've updated the description and title.

**Comment by kimrutherford** [comment]
Now I've looked a bit more closely, there are some duplicate alleles but most of the problem is duplicate genotypes.  I've made a separate issue about that: #624

**Comment by kimrutherford** [comment]
I think I've found another reason we have duplicate alleles.  When loading an allele from a PHAF file the loader checks to see if there is an allele with the same name in Chado.  Some PHAF files have "<systematic_id>delta" as the allele name (eg. SPBC1271.11delta) and we were looking for that.  But in Chado we always have "<gene_name>delta" if the allele comes from the contig files or from Canto.  So the loader was failing to see that the allele was there already.

I've changed the PHAF loader so that it always looks for "<gene_name>delta" if the allele is a deletion.

I've killed the nightly load and re-started it with the fix.  It should finish at lunchtime or so.
